File: Client/app/model/employee.ts
import { Pager } from './pager';
export class Employee {
    employeeId: number;
    firstName: string;
    lastName: string;
    jobTitle: string;
    hireDate: Date;
    department: string;
    email: string;
    created: Date;
    Modified: Date;
}

export class EmployeeList extends Pager
{	
	result: Employee[];
}

File: Model/Employee.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Threading.Tasks;

namespace app3.Model
{
    public class Employee
    {
        [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
        public int EmployeeId { get; set; }
        public string FirstName { get; set; }
        public string LastName { get; set; }
        public string JobTitle { get; set; }
        public DateTime HireDate { get; set; }
        public string Department { get; set; }
        public string Email { get; set; }

        public DateTime Created { get; set; }
        public DateTime Modified { get; set; }
    }
}
